setup.py

```diff
@@ -0,0 +1,32 @@
+import setuptools
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+setuptools.setup(
+    name="omexml-dls",
+    version="1.0.0",
+    author="Thomas M Fish",
+    author_email="thomas.fish@diamond.ac.uk",
+    description="Package for simple and consistent creation and parsing of OME metadata for B24 of Diamond Light Source Ltd.",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    url="https://github.com/DiamondLightSource/python-omexml-dls",
+    extras_require={
+        "test": [
+            "pytest"
+        ]
+    },
+    install_requires=[
+        "future",
+    ],
+    packages=["oxdls"],
```

Comment:
For a single module, see https://packaging.python.org/guides/distributing-packages-using-setuptools/#py-modules